lib/processor.js

```diff
@@ -90,9 +90,6 @@ function htmlToReactComponentsLib(tree, options) {
     moduleType = 'es6';
   }
 
-
-  var delimiter = options.moduleFileNameDelimiter || '-';
```

Comment:
To fix wrong filename when calling via API you may want to leave this, but change alternative value to empty string `''`.